config/src/main/resources/org/springframework/security/config/spring-security-5.1.xsd

```diff
@@ -2721,6 +2721,7 @@
          <xs:enumeration value="LOGIN_PAGE_FILTER"/>
          <xs:enumeration value="LOGOUT_PAGE_FILTER"/>
          <xs:enumeration value="DIGEST_AUTH_FILTER"/>
+         <xs:enumeration value="BEARER_TOKEN_AUTH_FILTER"/>
```

Comment:
Is this something that we should wait on until https://github.com/spring-projects/spring-security/issues/5480

Reply:
@jzheaux I'm not sure if the `xsd` or `rnc` file should have been changed? I never did update these for `oauth2-client` filters given that xml support is not there yet. Also, when I do a local build one of these files is changed which never happened before this commit. I believe it's the `xsd` file that get's generated. 

@rwinch Is this change necessary?